wearipedia/devices/fitbit/fitbit_sense_gen.py

```diff
@@ -389,8 +393,55 @@ def create_syn_data(start_date, end_date):
         full_dict["minutesLightlyActive"].append(activity[3])
         full_dict["distance"].append(activity[4])
         full_dict["minutesSedentary"].append(activity[5])
-        full_dict["heart_rate"].append(get_heart_rate(date))
+        full_dict["heart_rate_day"].append(get_heart_rate(date))
         full_dict["hrv"].append(get_hrv(date))
         full_dict["distance_day"].append(get_distance_day(date))
 
+    # encapsulate to match original data shape
```

Comment:
You can also collapse all of the following for loops into a double for loop. See comments in previous files on how

Reply:
Actually, all of this code might be unecessary. If the goal is just to have a dictionary of form {key: [{data_type: data}]}, in the "for date in synth_dates:" for loop above you could have just did append({data_type: data}) where data is something like activity[0].